corehq/apps/users/models.py

```diff
@@ -552,9 +552,11 @@ def add_domain_membership(self, domain, timezone=None, **kwargs):
 
     def add_as_web_user(self, domain, role, primary_location_id=None,
                         assigned_location_ids=None, program_id=None, profile=None,
-                        tableau_role=None, tableau_group_ids=None):
+                        tableau_role=None, tableau_group_ids=None, custom_user_data=None):
         if assigned_location_ids is None:
             assigned_location_ids = []
+        if custom_user_data is None:
```

Comment:
is this needed? It seems like the check on line 572 would take care of this and I don's see custom_user_data used anywhere else.

Reply:
you're right, that's not needed. [8405864](https://github.com/dimagi/commcare-hq/pull/35124/commits/840586437bdeea44cf652f2ef0cad18eeae92f0b)